### PR TITLE
[docs] Link to `vercel.com`

### PR DESCRIPTION
**Pull request by sarupbanskota**
We should add a link to vercel.com in the header section of the README — it'll direct readers to our website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
   <h3 align="center">Vercel</h3>
```
Comment (TooTallNate) [inline README.md]: Should we make this header the link instead?

**Comment by TooTallNate** [comment]
The readme in `packages/now-cli` should be updated as well.

File: README.md (hunk 1 above)
Comment (styfle) [reply README.md]: Agreed, we could make the name and image link to the website.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
   <p align="center">Develop. Preview. Ship.</p>
+  <p align="center"><a href="https://vercel.com">vercel.com</a></p>
```
Comment (styfle) [inline README.md]: Any changes here should also be made in `./packages/now-cli/README.md`

**Review by styfle — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -9,6 +11,8 @@
 
 ## Usage
 
+Vercel is the optimal workflow for frontend teams. All-in-one: Static and Jamstack deployment, Serverless Functions, and Global CDN.
+
```
Comment (styfle) [inline README.md:15]: Can you also add this line to the other README so the Usage sections match?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 <p align="center">
-  <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
-  <h3 align="center">Vercel</h3>
+  <a href="https://vercel.com">
+    <img src="https://assets.vercel.com/image/upload/v1588805858/repositories/vercel/logo.png" height="96">
+    <h3 align="center">Vercel</h3>
```
Comment (TooTallNate) [inline README.md:4]: Does `color` work?

```suggestion
    <h3 align="center" style="color: black;">Vercel</h3>
```

Comment (styfle) [reply README.md:4]: I have a feeling that arbitrary css doesn't work

Comment (TooTallNate) [reply README.md:4]: Ya, nvm. It's [not supported by GH](https://stackoverflow.com/q/11509830/376773). Also see: https://github.com/github/markup/issues/369

**Review by TooTallNate — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Pull request merged**